CSharp/DevVmPowershell/Helpers/Implementations/DisclaimerAcceptanceHelper.cs

```diff
@@ -56,7 +50,7 @@ public async Task AddDisclaimerConfigurationAsync(string workspaceName)
 		public async Task<bool> CheckIfDisclaimerConfigurationRdoExistsAsync(int workspaceId)
 		{
 			string url = Constants.Connection.RestUrlEndpoints.ObjectManager.QuerySlimUrl.Replace("-1", workspaceId.ToString());
-			HttpClient httpClient = RestHelper.GetHttpClient(InstanceAddress, AdminUsername, AdminPassword);
+			HttpClient httpClient = RestHelper.GetHttpClient(ConnectionHelper.RelativityInstanceName, ConnectionHelper.RelativityAdminUserName, ConnectionHelper.RelativityAdminPassword);
```

Comment:
In future, I would update the GetHttpClient to take only the ConnectHelper, instead of passing each property as a separate argument. Makes the code simpler I think. For now it's fine.